spec/features/polls/answers_spec.rb

```diff
@@ -61,7 +61,25 @@
     end
   end
 
-  pending "Update"
+  scenario "Update" do
+    answer = create(:poll_question_answer, question: question,
```

Comment:
It looks like this pull request was open before we had the useless assignment rule active :smile:.